using-list.py

```diff
@@ -0,0 +1,5 @@
+myStr = ['H','e','l','l','o',', ','W','o','r','l','d','!']
+
+for s in myStr:
+    print(s,end='')
+print()
```

Comment:
Remove the redundant print() and instead have the last line as an empty new line.
```suggestion

```

Reply:
Wow. I can't believe list iteration was yet to be implemented. Good find.

I'll be accepting this pull request as soon as you can make the requested changes.

Reply:
Doing it right away!